themes/luci-theme-bootstrap/luasrc/view/themes/bootstrap/header.htm

```diff
@@ -59,6 +59,16 @@ <h4><%:No password set!%></h4>
 				</div>
 			<%- end -%>
 
+			<%- if boardinfo.initramfs then -%>
```

Comment:
@dangowrt I belive it's no longer `boardinfo.iniramfs` but actually in `boardinfo.filesystem == "initramfs"`

Reply:
The correct attribute name is `rootfs_type` rather than filesystem though.